core/module_specifier.rs

```diff
@@ -66,11 +66,20 @@ impl ModuleSpecifier {
     self.0.as_str()
   }
 
+  pub fn resolve_import(
```

Comment:
reason: making it possible to manually set the curr_dir without changing every place where the original resolve import function is used.